migrations/20200213004252-change-date-column-names.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(t => {
+      return Promise.all([
```

Comment:
i ran into an issue when running this migration - because sequelize doesn't guarantee migrations occur in a transaction, it's possible to put your db in a bad intermediate state
- for now, undoing and re-running all migrations solves the problem, but this won't be a good idea down the line
- for reference, two potential solutions are:
1. migrate the data so new non-null columns won't break the migration (fixing the root problem)
2. have every migration have only 1 change (blanket solution for all schema changes)

Reply:
When I'm fixing these migrations, should I be updating this file? Or should I be creating a new migration?

Reply:
see comment below, tldr just update this file for this PR